tests/test_config.py

```diff
@@ -123,6 +123,27 @@ def test_group(self):
 
         self.assert_spellcheck('.source.yml', self.bad_words1 + self.bad_words3, groups=['group1'])
 
+    def test_no_sources(self):
+        """Test missing source match raises a ValueError."""
```

Comment:
I normally just surround variable names in single back ticks so I don't have to fill the dictionary with programming terms. The current Python source file rule only scans docstrings and has a user defined filter that ignores content surrounded in back ticks.